rhobs/alerting/data_plane/prometheus.component_alerts.yaml

```diff
@@ -0,0 +1,28 @@
+apiVersion: monitoring.coreos.com/v1
+kind: PrometheusRule
+metadata:
+  name: rhtap-component-alerting
+  labels:
+    tenant: rhtap
+spec:
+  groups:
+  - name: component_alerts
+    interval: 1m
+    rules:
+    - alert: ComponentDeletionErrors
+      expr: |
+        (increase(has_component_failed_deletion_total[1h]))
+        /
+        (increase(has_component_deletion_total[1h]))  > 0.05
+      for: 5m
+      labels:
+        severity: warning
+        slo: true
+      annotations:
```

Comment:
I think we can add the actual query value to the summary/description. You can use `{{ $value }}` for that.